ocs_ci/ocs/node.py

```diff
@@ -1686,3 +1686,102 @@ def get_node_mon_ids(node_name):
     mon_pods = pod.get_mon_pods()
     node_mon_pods = get_node_pods(node_name, pods_to_search=mon_pods)
     return [pod.get_mon_pod_id(mon_pod) for mon_pod in node_mon_pods]
+
+
+def get_nodes_where_ocs_pods_running():
+    """
+    Get the node names where rook ceph pods are running
+
+    Returns:
+        set: node names where rook ceph pods are running
+
+    """
+    pods_openshift_storage = pod.get_all_pods(
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+    )
+    ocs_nodes = list()
+    for pod_obj in pods_openshift_storage:
+        if (
+            "rook-ceph" in pod_obj.name
+            and "rook-ceph-operator" not in pod_obj.name
+            and "rook-ceph-tool" not in pod_obj.name
+        ):
+            try:
+                ocs_nodes.append(pod_obj.data["spec"]["nodeName"])
+            except Exception as e:
+                log.info(e)
+    return set(ocs_nodes)
+
+
+def get_node_rack(node_type=constants.WORKER_MACHINE):
+    """
+    Get node rack
+
+    Args:
+        node_type (str): The node type (e.g. worker, master)
```

Comment:
Rack labels are added only to worker nodes

Reply:
done